serpent-moltres/fuel-column-sensitivity/postprocessing.py

```diff
@@ -0,0 +1,406 @@
+import numpy as np
```

Comment:
You already know what I'm going to say about this file. Please make a postprocessing script or class appropriate for use across all of these folders. Make the appropriate unique configurations in the folders for each simulation, but do not copy the commonly useful functions in this file across every folder in your repo.